bin.js

```diff
@@ -27,19 +27,34 @@ const yargs = require('yargs')
         describe: 'The output file to create, - for stdout',
         default: '-'
       })
+      .option('quality', {
+        alias: 'q',
+        describe: 'The jpeg compression quality, between 0 and 1',
+        default: '1',
+        type: 'number'
+      })
       .option('images', {
         alias: 'm',
         type: 'array',
         describe: 'Which images to decode, -1 for all',
         default: [0]
+      })
+      .check((args) => {
+        if (args.quality != null) {
+          if (args.quality <= 0 || args.quality > 1) {
+            throw 'quality must be a number between 0 and 1';
```

Comment:
All values being thrown should be errors. This might even be a time where it's appropriate to use `RangeError`!

```suggestion
            throw new RangeError('quality must be a number between 0 and 1');
```